clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/services/ContainerInformationService.java

```diff
@@ -95,18 +101,41 @@ public ContainerInformationService(
 
     // Task-based health
     if (task != null) {
+      String taskDefinitionCacheKey =
+          Keys.getTaskDefinitionKey(accountName, region, service.getTaskDefinition());
+      TaskDefinition taskDefinition = taskDefinitionCacheClient.get(taskDefinitionCacheKey);
```

Comment:
Check for the task definition being null

Reply:
Whoops, don't know how I missed this! Good catch!